base/special/exp.jl

```diff
@@ -83,19 +96,19 @@ end
 end
 
 @inline function expb_kernel(::Val{2}, x::Float32)
-    return evalpoly(x, (1.0f0, 0.6931472f0, 0.2402265f0,
-                        0.05550411f0, 0.009618025f0,
-                        0.0013333423f0, 0.00015469732f0, 1.5316464f-5))
+    return evalpoly_ps8(x, (1.0f0, 0.6931472f0, 0.2402265f0,
```

Comment:
These are degree 7, no?

Reply:
Yes. Thanks. Fixed. I will provide some comparisons for the kernel functions as well to see if it makes sense to add 0 coeffs like I did now.